src/Aspire.Dashboard/Components/Dialogs/TextVisualizerDialog.razor

```diff
@@ -0,0 +1,77 @@
+                              aria-label="@Loc[nameof(Dialogs.TextVisualizerSelectFormatType)]"
+                              OnMenuChanged="@OnFormatOptionChanged">
+                @foreach (var option in _options)
+                {
+                    <FluentMenuItem Id="@option.Id" Disabled="@(!EnabledOptions.Contains(option.Id))" Value="@option.Id">@option.Name</FluentMenuItem>
+                }
+            </FluentMenuButton>
+        </div>
+    </FluentStack>
+</FluentDialogHeader>
+
+<FluentDialogBody>
+    <div class="log-overflow" style="max-height: 60vh; margin-bottom: 10px;">
+        <div class="log-container" id="@_logContainerId">
+            <Virtualize Items="@FormattedLines" ItemSize="20" OverscanCount="100">
+                <div class="log-line-row-container">
+                    <div class="log-line-row">
+                        <span class="log-line-area">
+                            <span class="log-line-number text-visualizer-line-number">@context.LineNumber</span>
+                            @if (context.IsFormatted)
+                            {
+                                // data-content and data-language used in javascript when re-highlighting a node when its
+                                // content changes (through scrolling)
+                                <span class="@GetLogContentClass()" data-content="@context.Content" data-language="@FormatKind">
+                                    @context.Content
+                                </span>
+                            }
+                            else
+                            {
+                                <span class="log-content">
+                                    @context.Content
+                                </span>
+                            }
+                        </span>
+                    </div>
+                </div>
+            </Virtualize>
+        </div    </div>
+
+    <FluentButton
+        Id="@_copyButtonId"
+        Style="float: right"
+        AdditionalAttributes="@FluentUIExtensions.GetClipboardCopyAdditionalAttributes(FormattedText, ControlsStringsLoc[nameof(ControlsStrings.GridValueCopyToClipboard)], ControlsStringsLoc[nameof(ControlsStrings.GridValueCopied)])">
+                    <span slot="start">
+                        <FluentIcon Class="copy-icon" Style="display:inline; vertical-align: text-bottom" Icon="Icons.Regular.Size16.Copy" Slot="start" />
+                        <FluentIcon Class="checkmark-icon" Style="display:none; vertical-align: text-bottom" Icon="Icons.Regular.Size16.Checkmark" Slot="start" />
+                    </span>
+        @ControlsStringsLoc[nameof(ControlsStrings.GridValueCopyToClipboard)]
+    </FluentButton>
+</FluentDialogBody>
+
+<FluentDialogFooter Visible="false" />
+
+@code {
+    [Parameter, EditorRequired]
+    public required TextVisualizerDialogViewModel Content { get; set; }
```

Comment:
Is there a reason to put the parameter in the razor file and not in the cs file? I've seen them in a number of razor files.

Code in a cs file offers better tooling experience compared to razor which is why I think it's better..

Reply:
No reason. Moved